scripts/db.js

```
@@ -1,78 +1,214 @@
+
+  async init() {
+    try {
+      const path_userData = app.getPath("userData");
+      const filePath = path.join(path_userData, "db_login.json");
+      const data = JSON.parse(fs.readFileSync(filePath, { encoding: "utf8" }));
+      this.#connection_data.host = "localhost";
+      this.#connection_data.user = data.user;
+      this.#connection_data.password = data.password;
+      this.#connection_data.database = data.database;
+    } catch (e) {
+      throw e;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)